src/goose/cli/main.py

```diff
@@ -17,8 +17,8 @@ def goose_cli() -> None:
     pass
 
 
-@goose_cli.command()
-def version() -> None:
+@goose_cli.command("version")
```

Comment:
[nit] `@goose_cli.command(name="version")`

Reply:
good catch! added in 295f634